test/Libraries/Microsoft.Extensions.DataIngestion.Tests/Readers/MarkItDownReaderTests.cs

```diff
@@ -40,7 +38,7 @@ protected override void SimpleAsserts(IngestionDocument document, string source,
     // The original purpose of the MarkItDown library was to support text-only LLMs.
     // Source: https://github.com/microsoft/markitdown/issues/56#issuecomment-2546357264
     // It can extract images, but the support is limited to some formats like docx.
-    [ConditionalFact]
+    [ConditionalFact(nameof(MarkItDownCondition.IsMarkItDownInstalled))]
```

Comment:
The `[ConditionalClass]` annotation already provides the condition.

```suggestion
    [ConditionalFact]
```